src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/effect/BurnoutReduction.java

```diff
@@ -2,16 +2,32 @@
 
 import com.github.minecraftschurlimods.arsmagicalegacy.common.init.AMAttributes;
 import net.minecraft.world.effect.MobEffectCategory;
+import net.minecraft.world.effect.MobEffectInstance;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.ai.attributes.AttributeInstance;
 import net.minecraft.world.entity.ai.attributes.AttributeModifier;
-import net.minecraft.world.entity.ai.attributes.RangedAttribute;
 
+import java.util.Objects;
+
+// Special handling is needed as calling addAttributeModifier(AMAtrributes.BURNOUT_REGEN.get(), ...) causes errors during registration
```

Comment:
this should not be happening check on forge discord why it could be happening and revert to addAttributeModifier (we should not be doing the modifier stuff manually)